package.json

```diff
@@ -1,40 +1,33 @@
 {
-  "name": "byu-jwt",
-  "version": "3.1.2",
-  "description": "The byu-jwt module provides helpful functions to retrieve a specified BYU .well-known URL and verify BYU signed JWTs.",
-  "main": "index.js",
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/byu-oit/byu-jwt-nodejs.git"
-  },
-  "author": "Gary Crye",
-  "license": "Apache-2.0",
-  "bugs": {
-    "url": "https://github.com/byu-oit/byu-jwt-nodejs/issues"
+  "name": "root",
+  "private": true,
+  "engines": {
+    "npm": ">=9",
+    "node": ">=18"
   },
   "scripts": {
-    "test": "npx nyc --reporter=lcov --reporter=text-summary mocha test --exit",
-    "lint": "npx standard"
-  },
-  "engines": {
-    "node": ">=8"
+    "build": "npx lerna run build",
+    "clean": "npx lerna run clean && nx reset",
+    "docs": "typedoc",
+    "lint": "npm run build && npx eslint .",
+    "lint:fix": "npm run build && npx eslint . --fix",
+    "test": "npx lerna run test",
+    "test:coverage": "npx lerna run test:coverage"
   },
-  "files": [
-    "lib",
-    "index.js",
-    "index.d.ts"
+  "workspaces": [
+    "packages/*"
   ],
-  "homepage": "https://github.com/byu-oit/byu-jwt-nodejs#readme",
-  "dependencies": {
-    "debug": "^4.1.1",
-    "jsonwebtoken": "^8.5.1"
-  },
   "devDependencies": {
```

Comment:
We should add `eslint` as a devDependency. We're using it in scripts and we have ESLint config files.

Reply:
I've gone back and forth about including eslint in each of the packages or just at the root. I'm now leaning toward just having it in the root and removing the liny scripts from the individual packages.

Reply:
✅